### PR TITLE
Adding API changes to the change log due to PRs 99 and 102.

### PR DESCRIPTION
**Pull request by kmacdonald-stsci**
Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)
Resolves [RCAL-nnnn](https://jira.stsci.edu/browse/RCAL-nnnn)

Closes #

This PR addresses API changes due to PRs #99 and #102.  The change log has now been updated to reflect those API changes.

**Checklist**
- [ ] added entry in `CHANGES.rst` (either in `Bug Fixes` or `Changes to API`)
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] updated relevant milestone(s)
- [ ] added relevant label(s)

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -23,6 +23,12 @@ saturation
 Changes to API
 --------------
```
Comment (zacharyburnett) [inline CHANGES.rst:25]:
```suggestion

ramp_fitting
~~~~~~~~~~~~
```

**Review by zacharyburnett — CHANGES_REQUESTED** [review]

**Review by zacharyburnett — APPROVED** [review]

**Comment by zacharyburnett** [comment]
one thing, could you also remove the entries from the `Bug Fixes`

File: CHANGES.rst [hunk 2]
```diff
@@ -23,6 +23,15 @@ saturation
...
 
+ramp_fitting
+~~~~~~~~~~~~
+
+- The tuple ``integ_info`` longer returns ``int_times`` as a part of it,
```
[the remark this answers is not in the document]
Comment (kmacdonald-stsci) [reply CHANGES.rst]: Fixed.

**Comment by zacharyburnett** [comment]
I apologize, I wasn't entirely clear, I meant remove the entries #99 and #102 from the `Bug Fixes` section, not remove the section entirely. This should just be a move of these entries from `Bug Fixes` to `Changes to API`

**Comment by kmacdonald-stsci** [comment]
> I apologize, I wasn't entirely clear, I meant remove the entries #99 and #102 from the `Bug Fixes` section, not remove the section entirely. This should just be a move of these entries from `Bug Fixes` to `Changes to API`

Those PRs were bug fixes that changed the API.  How should these be put into the change log?  It seems like both sections should be updated.

**Comment by zacharyburnett** [comment]
I'm not sure, I will ask @nden

**Pull request merged**